### PR TITLE
Raid clue scroll marker

### LINKED ISSUE #3419: Clues from raid loot
Add a clue scroll emoji before the spoiler on loot from cox/tob if you get a clue (elite/gm on bso), like the purple square if you get a unique

### PR DESCRIPTION
**Pull request by ianfights**
I added a clue scroll emoji and now when a user receives a clue scroll from a raid it will pop up with the emoji except when there is an important drop (as the user is most likely going to be more excited over the purple)

image

This should fix  #3419 

-   [x] I have tested all my changes thoroughly.

### CODE REVIEW

File: src/lib/constants.ts [hunk 1]
```diff
@@ -119,6 +119,7 @@ export const enum Emoji {
 	Ranged = '<:ranged:630911040258834473>',
 	Gear = '<:gear:835314891950129202>',
 	Slayer = '<:slayer:630911040560824330>',
+	Clue = '<:Clue_scroll:365003979840552960>',
```
Comment (Lajnux) [inline src/lib/constants.ts:122]:
```suggestion
	Clue = '<:clue_scroll:365003979840552960>',
```
small letter for consistency.

**Comment by Lajnux** [comment]
Are you planning to resolve the conflict? looks good to merge if resolved.

**Comment by ianfights** [comment]
> Are you planning to resolve the conflict? looks good to merge if resolved.

Yep I'll go ahead and make those changes once I get home

**Comment by ianfights** [comment]
Conflicts should be fixed

**Comment by Lajnux** [comment]
run a yarn lint and it should all be good.

**Comment by ianfights** [comment]
Adressed the issue

File: src/tasks/minions/minigames/raidsActivity.ts [hunk 2]
```diff
@@ -88,8 +88,10 @@ export const raidsTask: MinionTask = {
 			}
 			const str = isPurple ? `${emote} ||${itemsAdded}||` : itemsAdded.toString();
 			const deathStr = deaths === 0 ? '' : new Array(deaths).fill(Emoji.Skull).join(' ');
-
-			resultMessage += `\n${deathStr} **${user}** received: ${str} (${personalPoints?.toLocaleString()} pts, ${
+			const clue = items.some(([item]) => item.name.toLowerCase().includes('clue scroll'));
```
Comment (gc) [inline src/tasks/minions/minigames/raidsActivity.ts:91]: `if (ClueTiers.some(t => items.has(t.scrollID))` (rather than arbitrary string checking)

**Review by gc — CHANGES_REQUESTED** [review]

File: src/tasks/minions/minigames/raidsActivity.ts (hunk 2 above)
Comment (osrsjobbo) [reply src/tasks/minions/minigames/raidsActivity.ts]: Done

**Pull request closed without merging**